controllers/likedVideos.controller.js

```diff
@@ -0,0 +1,53 @@
+const { User } = require("../models/user.model");
+
+const getLikedVideos = async (req, res) => {
+  try {
+    const result = await User.findOne({}).populate("likedVideos.videoId");
+    res.status(200).json({ success: true, likedVideos: result.likedVideos });
+  } catch (err) {
+    res.status(500).json({
+      success: false,
+      message: "Could not fetch your cart",
+      errMessage: err.message,
+    });
+  }
+};
+
+const addVideoToLikedVideos = async (req, res) => {
+  try {
+    const userAndVideoDetails = req.body;
+
+    const result = await User.findOne({ _id: userAndVideoDetails.userId });
```

Comment:
You can directly find user using findById(userId) here

Reply:
Right! Will change that